Sources/ClientRuntime/Telemetry/DefaultTelemetry.swift

```diff
@@ -0,0 +1,151 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Namespace for the Default SDK Telemetry implementations.
+public enum DefaultTelemetry {
+    /// The Default SDK Telemetry Provider Implementation.
+    ///
+    /// - contextManager: no-op
+    /// - loggerProvider: provides SwiftLoggers
+    /// - meterProvider: no-op
+    /// - tracerProvider: no-op
+    public static let Provider: TelemetryProvider = _DefaultTelemetryProvider()
+
+    fileprivate class _DefaultTelemetryProvider: TelemetryProvider {
+        let contextManager: TelemetryContextManager = DefaultTelemetryContextManager
+        let loggerProvider: LoggerProvider = DefaultLoggerProvider
+        let meterProvider: MeterProvider = DefaultMeterProvider
+        let tracerProvider: TracerProvider = DefaultTracerProvider
+    }
+}
+
+// Context
+extension DefaultTelemetry {
+    fileprivate static let DefaultTelemetryContextManager: TelemetryContextManager = NoOpTelemetryContextManager()
+    fileprivate static let DefaultTelemetryContext: TelemetryContext = NoOpTelemetryContext()
+    fileprivate static let DefaultTelemetryScope: TelemetryScope = NoOpTelemetryScope()
+
+    fileprivate class NoOpTelemetryContextManager: TelemetryContextManager {
+        func current() -> TelemetryContext { DefaultTelemetryContext }
+    }
+    fileprivate class NoOpTelemetryContext: TelemetryContext {
+        func makeCurrent() -> TelemetryScope { DefaultTelemetryScope }
```

Comment:
Q: What are some things an actual (non no-op) implementation of `makeCurrent()` would do?

Reply:
Examples include managing concurrency or other distributed states.